src/main/java/edu/harvard/iq/dataverse/ingest/IngestMessageBean.java

```diff
@@ -111,6 +113,7 @@ public void onMessage(Message message) {
                         // and "mixed success and failure" emails. Now we never list successfully
                         // ingested files so this line is commented out.
                         // sbIngestedFiles.append(String.format("<li>%s</li>", datafile.getCurrentName()));
+                        indexService.indexDataset(datafile.getOwner(), true);
```

Comment:
I'd suggest using asyncIndexDataset - that will put the indexing in a queue and if multiple ingests complete at nearly the same time indexing will only get done once (fewer times anyway).

Reply:
Sure, done in b6f484c.